easybuild/easyconfigs/t/THOP/THOP-0.1.1.post2209072238-foss-2023a-PyTorch-2.1.2-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonBundle'
+
+name = 'THOP'
+version = '0.1.1.post2209072238'
+local_torch_version = '2.1.2'
+versionsuffix = '-PyTorch-' + local_torch_version + '-CUDA-%(cudaver)s'
```

Comment:
`THOP-0.1.1.post2209072238-foss-2023a.eb` does not have a versionsuffix on PyTorch, please remove